rpcs3/rpcs3qt/emu_settings.cpp

```diff
@@ -611,22 +612,41 @@ void emu_settings::EnhanceRadioButton(QButtonGroup* button_group, emu_settings_t
 		return;
 	}
 
+	bool found = false;
+	int def_pos = -1;
+
 	for (int i = 0; i < options.count(); i++)
 	{
 		const QString localized_setting = GetLocalizedSetting(options[i], type, i);
 
 		button_group->button(i)->setText(localized_setting);
 
-		if (options[i] == selected)
+		if (!found && options[i] == selected)
 		{
+			found = true;
 			button_group->button(i)->setChecked(true);
 		}
+		else if (def_pos == -1 && options[i] == def)
+		{
+			def_pos = i;
+		}
 
 		connect(button_group->button(i), &QAbstractButton::clicked, [=, this]()
 		{
 			SetSetting(type, sstr(options[i]));
 		});
 	}
+
+	if (!found)
+	{
+		ensure(def_pos >= 0);
+
+		cfg_log.error("EnhanceRadioButton'%s' tried to set an invalid value: %s. Setting to default: %s.", cfg_adapter::get_setting_name(type), sstr(selected), sstr(def));
```

Comment:
sorry, I made a typo during copy pasta
```suggestion
		cfg_log.error("EnhanceRadioButton '%s' tried to set an invalid value: %s. Setting to default: %s.", cfg_adapter::get_setting_name(type), sstr(selected), sstr(def));
```